apps/dotcom/sync-worker/src/getFetchEverythingSql.ts

```diff
@@ -45,7 +45,7 @@ export function getFetchUserDataSql(userId: string) {
 	return sql`
 SELECT 'user' AS "table", null::text as "lsn", ${sql.raw(userColumns + ',' + fileNulls + ',' + fileStateNulls)} FROM public.user WHERE "id" = '${sql.raw(userId)}'
 UNION
-SELECT 'file' AS "table", null::text as "lsn", ${sql.raw(userNulls + ',' + fileColumns + ',' + fileStateNulls)} FROM public.file WHERE "isDeleted" = false AND ("ownerId" = '${sql.raw(userId)}' OR "shared" = true AND EXISTS(SELECT 1 FROM public.file_state WHERE "userId" = '${sql.raw(userId)}' AND public.file_state."fileId" = public.file.id))
+SELECT 'file' AS "table", null::text as "lsn", ${sql.raw(userNulls + ',' + fileColumns + ',' + fileStateNulls)} FROM public.file WHERE ("ownerId" = '${sql.raw(userId)}' OR "shared" = true AND EXISTS(SELECT 1 FROM public.file_state WHERE "userId" = '${sql.raw(userId)}' AND public.file_state."fileId" = public.file.id))
```

Comment:
@MitjaBezensek I reverted this change from the other day because i needed to undelete a file for someone recently and it was a pain to figure out how to do it. After this change and the bits above in TldrawApp.ts we will be able to undelete files for people by just toggling 'isDeleted' in the backend. Yes it will mean sending people useless old deleted file records but that's a pretty low overhead for the convenience I think. With zero we should be able to solve this cleanly.

Reply:
Sounds good. We do filter them out in the zero polyfill so they do not affect the max files calculation, so all good there.